tests/gold_tests/headers/normalize_ae.test.py

```diff
@@ -98,12 +90,12 @@ def allAEHdrs(shouldWaitForUServer, shouldWaitForTs, ts, host):
 
     if shouldWaitForUServer:
         # wait for the micro server
-        tr.Processes.Default.StartBefore(server, ready=CheckPort(server.Variables.Port))
+        tr.Processes.Default.StartBefore(server)
```

Comment:
Don't you want to add the WhenPort open check here too to replace the ready=CheckPort?

Reply:
no.. This is already set by the testing framework. Part of the fixes here are that we can use the When.PortReady logic and "now" it does it like netstat

Reply:
Oh .. so we can do another clean up and remove the other port open checks as well. I will do that in a different PR